### ISSUE TITLE
请问您会开源训练代码吗？

### ISSUE THREAD

**Issue by zouzhekang** [ali-vilab/UniAnimate#39]
Can you open source training code?

**Comment by wangxiang1230** [comment]
> Can you open source training code?

Hi, thanks for your attention. Due to corporate disclosure requirements, we have no plans to open source the training code. however, you can refer to this loss function (https://github.com/ali-vilab/UniAnimate/blob/549ee5fad7618500790929b0ae73151d36649045/tools/modules/diffusions/diffusion_ddim.py#L381)
to train your own models.
